docs/guide/data_types/maps.rst

```diff
@@ -283,6 +283,17 @@ elsewhere, you can use either of the following::
 Python makes use of pointers so if you want to alter the data and keep the
 original data in the map intact make sure to copy it.
 
+To create a complete copy of a Map object that is entirely independent of the original,
+use the built-in `copy https://docs.python.org/3/library/copy.html`__ module's deepcopy
```

Comment:
Can we use intersphinx mapping to link to the docs for the `copy` module in the standard library? I think something like

```suggestion
use the built-in `copy.deepcopy`
```

should work.